### ISSUE TITLE
Getting raw data for play by play

### ISSUE THREAD

**Issue by mesee298** [BurntSushi/nflgame#144]
First I have to state that Fantasy Football Coaches are more high maintenance than women. :)

Is there a way to get the raw Json data with what NFL numbers correlate to your variables a play by play information?

So in this NFL Play by Play (7:36) 22-C.Anderson up the middle to DEN 5 for no gain (50-J.Houston), I'm assuming that J.Houston has a QB Hit, but can tell without the json information.

**Comment by BurntSushi** [comment]
I'm not sure I understand the question. Can you rephrase it and preferably include some code to show what you've tried?

**Comment by mesee298** [comment]
Since QB HiTs don't show up on play by play, but it shows up in the json, how can I find what play by play that QB Hit occurred on?

**Comment by ochawkeye** [comment]
The stat type you are looking for is `defense_qbhit`, though I'm not positive what you are looking for.  Here's a guess

```python
import nflgame
games = nflgame.games(year=2015, week=2)
plays = nflgame.combine_plays(games).filter(defense_qbhit__ge=1)

for play in plays:
	print play
```

```
(KC, KC 29, Q1, 1 and 15) (10:50) A.Smith pass short left to J.O'Shaughnessy to DEN 47 for 24 yards (D.Stewart) [D.Ware]. PENALTY on DEN-D.Ware, Roughing the Passer, 15 yards, enforced at DEN 47.
(DEN, DEN 9, Q1, 1 and 10) (8:12) P.Manning sacked at DEN 5 for -4 yards (M.Devito).
(KC, KC 30, Q1, 2 and 7) (5:52) (Shotgun) A.Smith pass short middle to T.Kelce to KC 45 for 15 yards (T.Ward) [D.Ware].
(KC, KC 40, Q1, 1 and 15) (4:45) (Shotgun) A.Smith sacked at KC 36 for -4 yards (D.Ware).
(KC, KC 36, Q1, 2 and 19) (3:58) (Shotgun) A.Smith sacked at KC 31 for -5 yards (V.Miller). Penalty on KC-B.Grubbs, Offensive Holding, declined.
(DEN, DEN 11, Q1, 1 and 10) (3:21) P.Manning pass incomplete deep left to E.Sanders (M.Peters) [T.Hali].
(DEN, DEN 33, Q1, 3 and 4) (:37) P.Manning sacked at DEN 25 for -8 yards (J.Houston).
(DEN, DEN 46, Q3, 3 and 13) (7:13) P.Manning pass short left to D.Thomas to KC 45 for 9 yards (P.Gaines; J.Fleming) [T.Hali].
(DEN, DEN 26, Q4, 2 and 10) (14:56) (No Huddle, Shotgun) P.Manning pass incomplete deep right to D.Thomas [D.Ford].
(KC, KC 30, Q4, 3 and 10) (12:18) (Shotgun) A.Smith sacked at KC 22 for -8 yards (A.Smith).
(DEN, DEN 23, Q4, 3 and 9) (10:44) (Shotgun) P.Manning sacked at DEN 17 for -6 yards (J.Houston).
(KC, KC 40, Q4, 2 and 9) (4:08) (Shotgun) A.Smith pass short right to T.Kelce to DEN 31 for 29 yards (D.Stewart) [D.Ware].
(DEN, KC 19, Q4, 1 and 10) (:49) (Shotgun) P.Manning pass incomplete deep left to A.Caldwell [J.Houston].
(KC, KC 31, Q4, 1 and 10) (:19) (Shotgun) A.Smith sacked at KC 27 for -4 yards (M.Jackson).
```

**Comment by mesee298** [comment]
So on the following play by play (from your example) how do I know who did the QBHit?  Is it P.Gaines, J.Fleming or T.Hali? 

(DEN, DEN 46, Q3, 3 and 13) (7:13) P.Manning pass short left to D.Thomas to KC 45 for 9 yards (P.Gaines; J.Fleming) [T.Hali].

**Comment by ochawkeye** [comment]
```python
import nflgame
games = nflgame.games(year=2015, week=2, home='DEN', away='DEN')
plays = nflgame.combine_plays(games).filter(defense_qbhit__ge=1)

for play in plays:
	print play
	for player in play.players.filter(defense_qbhit__ge=1):
		print player.player, 'hit the quarterback'
	print '-'
```
```
(KC, KC 29, Q1, 1 and 15) (10:50) A.Smith pass short left to J.O'Shaughnessy to DEN 47 for 24 yards (D.Stewart) [D.Ware]. PENALTY on DEN-D.Ware, Roughing the Passer, 15 yards, enforced at DEN 47.
DeMarcus Ware (OLB, DEN) hit the quarterback
-
...
-
(KC, KC 31, Q4, 1 and 10) (:19) (Shotgun) A.Smith sacked at KC 27 for -4 yards (M.Jackson).
Malik Jackson (DE, DEN) hit the quarterback
-
```

**Comment by mesee298** [comment]
Thanks for your help on this.
